java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -247,77 +252,17 @@ private BazelWorkspaceDriverException(String message) {
     }
   }
 
-  /**
-   * bazel command with the given arguments. features like bazelrc file and environment can be added
-   * to result object
-   */
-  public BazelCommand bazelCommand(String arg, String... args) {
-    return bazelCommand(
-        Stream.concat(Stream.of(arg), Stream.of(args)).collect(Collectors.toList()));
-  }
-
-  public BazelCommand bazelCommand(List<String> args) {
-    if (currentBazel == null) {
-      throw new BazelWorkspaceDriverException(
-          "Cannot use bazel because no version was specified, "
-              + "please call bazelVersion(version) before creating command(...).");
-    }
-    return new BazelCommand(args);
+  /** Returns a builder for invoking bazel. */
+  public BazelCommand.Builder bazel(String arg, String... args) {
```

Comment:
It seems like we are moving back and forth between "bazel" and "bazelCommand" for this function :) Anyway I prefer simply "bazel"